### ISSUE TITLE
TravisCI UnitTests wrong parameters

### ISSUE THREAD

**Issue by bumbummen99** [restcord/restcord#146]
Currently the tests are broken with `403 Forbidden` for all calls. Are the parameters passed to the phpunit command still valid and available?

**Comment by aequasi** [comment]
https://github.com/restcord/restcord/blob/master/bin/stupidFunctionalTest.php#L38

Requires passing in a token

**Comment by bumbummen99** [comment]
> 
> 
> https://github.com/restcord/restcord/blob/master/bin/stupidFunctionalTest.php#L38
> 
> Requires passing in a token

Yes but it seems liek they are not passed correctly in TravisCI, it seems the last time the tests ran past the redis step the `$token $guild_id $channel_id` passed caused the errors.
https://travis-ci.org/github/restcord/restcord/jobs/751300665
https://travis-ci.org/github/restcord/restcord/jobs/740482383

**Comment by cryptiklemur** [comment]
RestCord 0.9.0 removed Travis and the credential-dependent functional test. GitHub Actions now runs the PHPUnit and Redis suites without Discord guild, channel, or token parameters.
